bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -5722,17 +5726,22 @@ void RewriteInstance::rewriteFile() {
 
   // Write all allocatable sections - reloc-mode text is written here as well
   for (BinarySection &Section : BC->allocatableSections()) {
-    if (!Section.isFinalized() || !Section.getOutputData())
+    if (!Section.isFinalized() || !Section.getOutputData()) {
+      BC->outs() << "BOLT: new section is finalized or !getOutputData, skip " << Section.getName() << '\n';
```

Comment:
This looks like a debug statement, consider wrapping into LLVM_DEBUG() macro.

Reply:
Sure.

Just a some question: 

what is the difference between LLVM_DEBUG and Verbosity cases? 
Do we consider to enable automatically -debug when we increase verbosity to some level

Reply:
LLVM_DEBUG statements are only enabled in debug build mode, and only printed if either `-debug` or corresponding `-debug-only` is set at runtime. Verbosity controls the amount of logging in normal execution mode, controlled by `-v=N` switch. The two are orthogonal.

Reply:
Done.
Put as LLVM_DEBUG + VerboseLevel > 1